nodejs/packages/layer/src/wrapper.ts

```diff
@@ -22,7 +22,8 @@ import {
 import { getEnv } from '@opentelemetry/core';
 import { AwsLambdaInstrumentationConfig } from '@opentelemetry/instrumentation-aws-lambda';
 import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-proto';
-import { MeterProvider, MeterProviderOptions } from '@opentelemetry/sdk-metrics';
+import { MeterProvider, MeterProviderOptions, PeriodicExportingMetricReader } from '@opentelemetry/sdk-metrics';
+import { OTLPMetricExporter } from '@opentelemetry/exporter-metrics-otlp-grpc'
```

Comment:
I think this should be exporter from `exporter-metrics-otlp-proto`. That's what what is in package.json, and the tracer is using HTTP proto as well.

Reply:
what's the differences between the 2 packages?

Reply:
Both use protobuf as the data format, but `exporter-*-otlp-proto` uses HTTP as the protocol.

Reply:
@stevemao Do you have any objection to this suggestion?

Reply:
Can make the changes

Reply:
@stevemao below you said
> I'm not sure what I need to change. Please submit your own PR to address any issue. Cheers!

This is the change I was referring to.  Seems like a straightforward suggestion. If you have questions, you can also stop by the SIG meeting to discuss.